cloud-data-protection/src/components/layout.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { Container } from 'reactstrap';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
import { Container } from "reactstrap";
```